py/torch_tensorrt/_compile.py

```diff
@@ -237,22 +239,27 @@ def compile(
         )
         return compiled_fx_module
     elif target_ir == _IRType.dynamo:
+        if kwargs["mutable"]:
```

Comment:
Figure out what the best way to access this feature is, since it only seems to work for nn.Module and not a lower level frontend

Reply:
```py

# Sets a flag that says it needs to be compiled 
trt_mod = torch_tensorrt.MutableTorchTensorRTModule(mod, ...)

# compile on first run 
trt_mod.forward()

Reply:
RE: Serialization, think about how we serialize the state (do we need to recompile).

Reply:
No recompilation is needed. Saving implemented